Library/Homebrew/utils/shell.rb

```diff
@@ -124,5 +134,19 @@ def sh_quote(str)
       str.gsub!(/\n/, "'\n'")
       str
     end
+
+    sig { params(str: String).returns(String) }
+    def rc_quote(str)
```

Comment:
Could you refactor this a bit to share more code with `sh_quote` and `csh_quote`? All three methods are almost identical bar a line or two.

Thanks!

Reply:
Can we assume that Homebrew won’t ever have a ' in any printed file name? If so, `sh_quote` would even be sufficient.

Reply:
Fixed.